packages/components/src/Card/_docs/Card.stories.tsx

```diff
@@ -23,30 +28,32 @@ export const Playground: Story = {
     },
   },
 }
-
-export const Variants: Story = {
+export const Color: Story = {
```

Comment:
Update this to `Colors` and we should be hunky dory 🕺

Reply:
```suggestion
export const Colors: Story = {
```